lib/arrays.md

```diff
@@ -5,44 +5,54 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+	An Array is a data object that stores an ordered list of values. The index must be an integer. 
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+	An Array is very handy to use for lists or keeping information sorted. You wouldn't add list of student number, student name and there information into one big string but into an Array to keep it sorted. 
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+	I would call by the index of the array such as array[6] or array[8]. If the element in the array being retrieved doesn’t exist you’ll receive nill response. 
```

Comment:
In Ruby, do Arrays' indexes start at `1` or `0`? If I have an Array `a = [1, 2, 3]`, how would I reference the `2`?